telethon/tl/custom/file.py

```diff
@@ -18,20 +18,6 @@ class File:
     def __init__(self, media):
         self.media = media
 
-    @property
-    def id(self):
```

Comment:
Breaking change. Even if the file IDs are unusable, people may be using them to store / refer to files.